airflow/providers/mongo/hooks/mongo.py

```diff
@@ -90,15 +85,25 @@ def get_conn(self) -> MongoClient:
             options.update({"ssl_cert_reqs": CERT_NONE})
 
         self.client = MongoClient(self.uri, **options)
-
         return self.client
 
-    def close_conn(self) -> None:
-        """Closes connection"""
-        client = self.client
-        if client is not None:
-            client.close()
-            self.client = None
+    def create_uri(self) -> str:
```

Comment:
```suggestion
    def _create_uri(self) -> str:
```

This doesn’t need to be public, from what I can tell?